docs/my-website/docs/providers/huggingface.md

```diff
@@ -2,466 +2,363 @@ import Image from '@theme/IdealImage';
-</a>
+### Serverless Inference Providers
+You can check available models for an inference provider by going to [huggingface.co/models](https://huggingface.co/models), clicking the "Other" filter tab, and selecting your desired provider:
 
-You need to tell LiteLLM when you're calling Huggingface.
-This is done by adding the "huggingface/" prefix to `model`, example `completion(model="huggingface/<model_name>",...)`.
+![Filter models by Inference Provider](../../img/hf_filter_inference_providers.png)
 
-<Tabs>
-<TabItem value="serverless" label="Serverless Inference API">
+### Dedicated Inference Endpoints
+Refer to the [Inference Endpoints catalog](https://endpoints.huggingface.co/catalog) for a list of available models.
 
-By default, LiteLLM will assume a Hugging Face call follows the [Messages API](https://huggingface.co/docs/text-generation-inference/messages_api), which is fully compatible with the OpenAI Chat Completion API.
+## Usage
 
 <Tabs>
-<TabItem value="sdk" label="SDK">
-
-```python
-import os
-from litellm import completion
-
-# [OPTIONAL] set env var
-os.environ["HUGGINGFACE_API_KEY"] = "huggingface_api_key"
+<TabItem value="serverless" label="Serverless Inference Providers">
 
-messages = [{ "content": "There's a llama in my garden 😱 What should I do?","role": "user"}]
+### Authentication
+With a single Hugging Face token, you can access inference through multiple providers. Your calls are routed through Hugging Face and the usage is billed directly to your Hugging Face account at the standard provider API rates.
 
-# e.g. Call 'https://huggingface.co/meta-llama/Meta-Llama-3.1-8B-Instruct' from Serverless Inference API
-response = completion(
-    model="huggingface/meta-llama/Meta-Llama-3.1-8B-Instruct",
-    messages=[{ "content": "Hello, how are you?","role": "user"}],
-    stream=True
-)
+Simply set the `HF_TOKEN` environment variable with your Hugging Face token, you can create one here: https://huggingface.co/settings/tokens.
 
-print(response)
+```bash
+export HF_TOKEN="hf_xxxxxx"
 ```
-
-</TabItem>
-<TabItem value="proxy" label="PROXY">
-
-1. Add models to your config.yaml
-
-```yaml
-model_list:
-  - model_name: llama-3.1-8B-instruct
-    litellm_params:
-      model: huggingface/meta-llama/Meta-Llama-3.1-8B-Instruct
-      api_key: os.environ/HUGGINGFACE_API_KEY
+or alternatively, you can pass your Hugging Face token as a parameter:
+```python
+completion(..., api_key="hf_xxxxxx")
 ```
 
-2. Start the proxy
+### Getting Started
 
-```bash
-$ litellm --config /path/to/config.yaml --debug
+To use a Hugging Face model, specify both the provider and model you want to use in the following format:
 ```
+huggingface/<provider>/<model_id>
+```
+Where `<model_id>` is the Hugging Face model ID and `<provider>` is the inference provider.
+By default, if you don't specify a provider, LiteLLM will use the [HF Inference API](https://huggingface.co/docs/api-inference/en/index).
```

Comment:
Maybe add an example of this below?

Reply:
this is mostly for backward compat with the existing LiteLLM implementation, correct? cc @ishaan-jaff